Add conclusion (#33)
greenelab/xswap-manuscript: content/02.body.md

## Conclusion

Nodes in biomedical networks can have different degrees, and networks can have very different degree distributions.
Many edge prediction methods are correlated with degree.
Degree imbalance, in which edges are distributed unequally between nodes, can lead edge prediction methods to make nonspecific or trivial predictions.

> Sometimes networks faithfully represent all true relationships, but degree biases and other misrepresentations can obscure node degree.

Reviewer: ```suggestion
```
I think this paragraph is stronger without this sentence in it. Give it a read without it and see what you think.

> To better understand the effects of degree on an edge prediction task and on edge prediction methods, degree should be considered directly.

Reviewer: ```suggestion
Degree should be considered directly in prediction approaches to understand the effects degree has on edge prediction tasks and methods.
```

We developed a network permutation framework and open source software implementation that quantifies the probability of edge existence due to degree and can be used to assess the fraction of feature performance attributable to degree.
We demonstrated the superiority of the edge prior over other degree-based features for quantifying the effect of degree on the probability of edge existence.

> We provided an example application of our methods that assessed five edge prediction features' performances on a network reconstruction task by quantifying the methods' reliance on degree and degree's performance on the prediction task.

Reviewer: ```suggestion
```
I also think it's stronger without this sentence. Again, see what you think 😁

> Our method provides a context for evaluating edge prediction methods and specific predictions for reliance on degree and, therefore, nonspecificity.

Reviewer: ```suggestion
The XSwap methods and software provide a context for evaluating edge prediction methods and specific predictions for reliance on degree and, therefore, nonspecificity.
```
